packs/orion/requirements.txt

```diff
@@ -0,0 +1 @@
+orionsdk
```

Comment:
Probably a good idea to pin it to a more specific version in case the client API changes (e.g. `orionsdk>=0.0.3,<0.1`)